testing/tests/DevExpress.exporter/fileSaver.tests.js

```diff
@@ -91,26 +91,6 @@ QUnit.test("Save base64 via proxyUrl for IE < 10", function(assert) {
     }
 });
 
-QUnit.test("Save blob by _winJSBlobSave on winJS devices", function(assert) {
```

Comment:
This test checks nothing.  It just checks that internal method was called. I think this test must be improved in the context of another PR that it will check how this internal method works.